api/types/role_test.go

```diff
@@ -641,6 +641,49 @@ func TestUnmarshallCreateHostUserModeYAML(t *testing.T) {
 	}
 }
 
+func TestUnmarshallCreateDatabaseUserModeJSON(t *testing.T) {
+	for _, tc := range []struct {
+		expected CreateDatabaseUserMode
+		input    any
+	}{
+		{expected: CreateDatabaseUserMode_DB_USER_MODE_UNSPECIFIED, input: "\"\""},
```

Comment:
nit: put the input before the expected?

In Go it's also typical to talk about "got" vs "want" (but no need to change anything).